posthog/event_usage.py

```diff
@@ -41,8 +43,10 @@ def report_user_signed_up(
             props[f"org__{k}"] = v
 
     # TODO: This should be $set_once as user props.
-    posthoganalytics.identify(distinct_id, props)
-    posthoganalytics.capture(distinct_id, "user signed up", properties=props)
+    posthoganalytics.identify(user.distinct_id, props)
+    posthoganalytics.capture(
```

Comment:
Wonder why we don't use your new `report_user_action` method here

Reply:
Because of order I converted these in. No strong feelings - we've centralized most tracking here so it will be easy to change if ever needed.